geomstats/geometry/hyperbolic.py

```diff
@@ -361,19 +394,20 @@ def _half_plane_to_extrinsic_coordinates(point):
     def _extrinsic_to_half_plane_coordinates(point):
         """Convert extrinsic to half plane coordinates.
 
-        Convert the parameterization of a point on the Hyperbolic space
+        Convert the parameterization of a point on the hyperbolic plane
```

Comment:
"on the hyperbolic plane" -> "in the hyperbolic plane"

Reply:
Corrected.